src/utils/wallets/wallets.service.ts

```diff
@@ -127,6 +127,12 @@ async function fetchFirstAvailableAuthShare(
     for (const wallet of wallets) {
       const { id: walletId, deviceShare } = wallet;
 
+      if (deviceShare === null) {
```

Comment:
@pawanpaudel93 That issue we had where we would get an error in the console when trying to activate a wallet because its `deviceShare` was `null` happens when the device share has been somehow removed from `localStorage`. With this new check we won't get the error, and this case will be handled by the `return` at the end of this function.